File: laramie-tesoreria-webapi/src/domain/entities/cuenta-contable.ts
export default class CuentaContable {

    id: number;
	codigo: string;
	nombre: string;
	orden: number;
	agrupamiento: string;
	ejercicio: string;
    tipo: number;
	ejercicioAgrupamiento: string;
	porcentajeAfectada: number;
	conChequeRetencion: boolean;
	fechaBaja: Date;

	constructor(
        id: number = 0,
		codigo: string = "",
		nombre: string = "",
		orden: number = 0,
		agrupamiento: string = "",
		ejercicio: string = "",
		tipo: number = 0,
		ejercicioAgrupamiento: string = "",
		porcentajeAfectada: number = 0,
		conChequeRetencion: boolean = false,
		fechaBaja: Date  = null
	)
	{
        this.id = id;
		this.codigo = codigo;
		this.nombre = nombre;
		this.orden = orden;
		this.agrupamiento = agrupamiento;
		this.ejercicio = ejercicio;
		this.tipo = tipo;
		this.ejercicioAgrupamiento = ejercicioAgrupamiento;
		this.porcentajeAfectada = porcentajeAfectada;
		this.conChequeRetencion = conChequeRetencion;
		this.fechaBaja = fechaBaja;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.codigo = row.codigo ?? "";
		this.nombre = row.nombre ?? "";
		this.orden = row.orden ?? 0;
		this.agrupamiento = row.agrupamiento ?? "";
		this.ejercicio = row.ejercicio ?? "";
		this.tipo = row.tipo ?? 0;
		this.ejercicioAgrupamiento = row.ejercicioAgrupamiento ?? "";
		this.porcentajeAfectada = row.porcentajeAfectada ?? 0;
		this.conChequeRetencion = row.conChequeRetencion ?? false;
		this.fechaBaja = row.fechaBaja ?? null;
	}

}

File: modules/tesoreria/models/cuenta_contable.py
from sqlalchemy import Column, BigInteger, String, Integer, Date

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class CuentaContable(Base):
    __tablename__ = "tesoreria_cuentas_contables"

    id = Column(BigInteger, primary_key=True)
    codigo = Column(String(50), nullable=False)
    nombre = Column(String(250), nullable=False)
    orden = Column(Integer, nullable=False, default=0)
    agrupamiento = Column(String(50), nullable=True)
    ejercicio = Column(Integer, nullable=True)
    tipo = Column(String(50), nullable=True)
    fecha_baja = Column(Date, nullable=True)
